vtex-apps/render-runtime
# Fix exported types

#### What does this PR do? \*

Makes all TypeScript types from the app to work.

#### How to test it? \*

1. Go to this workspace: https://types--storecomponents.myvtex.com/
2. Link any app that uses a render-runtime feature
3. Delete any local vtex.render-runtime types 
3. Use `vtex setup --typings` to download the linked types
4. Check the amazing types working

![runtime](https://user-images.githubusercontent.com/284515/95630724-b67d5900-0a58-11eb-8a83-e50804bbfa9b.gif)

#### Describe alternatives you've considered, if any. \*



#### Related to / Depends on \*

n/a
